usaspending_api/awards/v2/views/transactions.py

```diff
@@ -0,0 +1,90 @@
+from copy import deepcopy
+
+from django.db.models import F
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from usaspending_api.awards.models import TransactionNormalized
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.helpers.generic_helper import get_simple_pagination_metadata
+from usaspending_api.common.views import APIDocumentationView
+from usaspending_api.core.validator.pagination import PAGINATION
+from usaspending_api.core.validator.tinyshield import TinyShield
+
+
+class TransactionViewSet(APIDocumentationView):
+    """
+    endpoint_doc: /awards/transactions.md
+    """
+    transaction_lookup = {
+        # "Display Name": "database_column"
+        "id": "transaction_unique_id",
+        "type": "type",
+        "type_description": "type_description",
+        "action_date": "action_date",
+        "action_type": "action_type",
+        "action_type_description": "action_type_description",
+        "modification_number": "modification_number",
+        "description": "description",
+        "federal_action_obligation": "federal_action_obligation",
+        "face_value_loan_guarantee": "face_value_loan_guarantee",
+        "original_loan_subsidy_cost": "original_loan_subsidy_cost",
+        # necessary columns which are only present for Django Mock Queries
+        "award_id": "award_id",
+        "is_fpds": "is_fpds",
+    }
+
+    def _parse_and_validate_request(self, request_dict: dict) -> dict:
+        models = deepcopy(PAGINATION)
+        models.append({"key": "award_id", "name": "award_id", "type": "integer",
+                      "optional": False, "default": None, "allow_nulls": True})
```

Comment:
Since `award_id` is required, I would remove the `default` and `allow_nulls` fields